steps/cadence-innovus-flowsetup/setup.tcl

```diff
@@ -401,6 +401,21 @@ set vars(route,restore_design,skip)             true
 set vars(postroute,restore_design,skip)         true
 set vars(signoff,restore_design,skip)           true
 
+# [Aug 2021] FAIL routeDesign immediately if placement is bad.
+# Also see garnet issue https://github.com/StanfordAHA/garnet/issues/803
+# 
+# ANECDOTE: without the "-placementCheck" switch, glb_tile CI build
+# went into postroute_hold with bad placement and was still runnning
+# after 122 hours (takes less than one hour with correct placement).
+# 
+# METHOD: Create and point to a tmp file containing the command that
+# will replace "routeDesign"
+
+set tmp_rrdrt $vars(custom_scripts_dir)/route,route_design,replace_tcl.tcl
+set tmpf [open $tmp_rrdrt w]
+puts $tmpf "routeDesign -placementCheck"; close $tmpf
+set vars(route,route_design,replace_tcl) $tmp_rrdrt
+
```

Comment:
This looks fine, but we should just actually create the custom script instead of making a temporary file, like we do here:

- https://github.com/mflowgen/mflowgen/blob/master/steps/cadence-innovus-flowsetup/innovus-foundation-flow/custom-scripts/restore-design.tcl

So can you create `cadence-innovus-flowsetup/innovus-foundation-flow/custom-scripts/route-design-check.tcl` and stick this command in there. Then all we need in this file is:

    set vars(route,route_design,replace_tcl) $vars(custom_scripts_dir)/route-design-check.tcl`

Reply:
Sure, no problem. I guess I was getting overzealous with the whole "single source of truth" thing.
I've made the changes, now I'm going to run a quick check to make sure nothing broke. Then hopefully
we can finalize the pull. Thanks again for all your help!